control_panel_api/tests/filters/test_s3bucket_filter.py

```diff
@@ -17,6 +17,12 @@ def setUp(self):
             "control_panel_api.S3Bucket", name="test-bucket-1")
         self.s3_bucket_2 = mommy.make(
             "control_panel_api.S3Bucket", name="test-bucket-2")
+        # Normal user has access to s3_bucket_1
```

Comment:
These comments aren't necessary. Intention can be deduced from name etc.  And the comments above also.